contrib/jemalloc/include/jemalloc/internal/atomic.h

```diff
@@ -81,6 +81,8 @@ JEMALLOC_GENERATE_INT_ATOMICS(uint32_t, u32, 2)
 JEMALLOC_GENERATE_INT_ATOMICS(uint64_t, u64, 3)
 #endif
 
+JEMALLOC_GENERATE_ATOMICS(uintptr_t, uptr, this_should_not_be_used)
```

Comment:
Hm, the use of `LG_SIZEOF_PTR` above for both `void *` and `(s)size_t` has alarm bells ringing in my head...

Reply:
I _think_ the third argument is not actually used, so it should be fine.